### ISSUE TITLE
Mispronunciations in Japanese TTS results / 日语TTS出现拼读错误

### ISSUE THREAD

**Issue by addicted0010** [FunAudioLLM/CosyVoice#303]
Locally deployed on my MACBookPro (M2 chip).
Results: 
[cozyvoice_results.zip](https://github.com/user-attachments/files/16685420/cozyvoice_results.zip)

1) Input text:
自民党は、岸田総理大臣の後任を**選ぶ**総裁選挙について、**9月**12日に告示し、**27日**に投開票を行うことを**決め**ました。総裁選挙の期間は、いまの**規程**が設けられて以降、最も**長く**なります。
Cozyvoice result:  audio1.wav

Issues:
There are 6 mispronunciations occurred in the result. I have marked with **BOLD** font.

2) Input text:
晴子、お前が見つけてきた変な男は、湘北に必要な男になったぞ
Cozyvoice result:  audio2.wav

Issues:
'晴子' was wrong and not sure what it said.
Only the middle part remains. The last part was missing.

3) Input text:
自分がバスケの試合をしている**時**も、そうでない**時**も、心が折れそうな自分を奮起させる最高の名言です。この**言葉**から、何度も力を**貰**いました。
Cozyvoice result:  audio3.wav

Issues:
4 mispronunciations occurred in the result. I have marked with **BOLD** font.

In conclusion, this result means cozyvoice is basically unusable for Japanese language.
The mispronunciation issue looks quite strange. These are all simple words in Japanese.
Is it caused by the polluted original Japanese training material?
Is there any improvement plan for Japanese capability?

*BTW, Chinese TTS result is great (wow!)

**Comment by aluminumbox** [comment]
emmm we use bpe tokenization, and this is robust only when training data is sufficient. currently we can only recommend you to train the llm model with more Japanese data, or you can try modify the token, for example 晴子 to 晴 子, or some other words that llm might have seen during training.

**Comment by lucasjinreal** [comment]
Hi, just wonder is there anyway to use '中文男' voice pronouce Japanese more right? currently, I set <|jp|> and using cn_male the japanese voice sound very weired.

I know there might lack of distribution, but is there anyway based on currently model?
